src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -100,19 +106,23 @@ private ResponseEntity postEventInternal(final String eventTypeName,
         }
     }
 
+    private void reportSLOs(final long startingNanos, final int totalSizeBytes, final int eventCount) {
+        final long msSpent = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startingNanos);
```

Comment:
I actually like that it is nanoseconds - that gives you better precision. 
`toMS(nano - nano)` gives better precision than `ms - ms`